app/helpers/fae/view_helper.rb

```diff
@@ -25,6 +25,17 @@ def fae_content_form(f, attribute, label: nil, hint: nil, helper_text: nil, mark
       render 'fae/application/content_form', f: f, attribute: attribute, label: label, hint: hint, helper_text: helper_text, markdown: markdown, markdown_supported: markdown_supported, input_options: input_options
     end
 
+    def fae_index_image(image, path = nil)
+      return if image.blank? || image.asset.blank? || image.asset.thumb.blank?
+      output = nil
+      if path
```

Comment:
I think what you're looking for is [`link_to_if`](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_if).